puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/rpm_packaging.groovy

```diff
@@ -67,14 +67,15 @@ pipeline {
                         def index = i
                         packages[packages_to_build[index]] = {
                             package_name = packages_to_build[index]
+                            spec_path = sh(returnStdout: true, script: "find -name \"${package_name}\\.spec\"").trim()
 
-                            new_version = query_rpmspec("packages/**/${package_name}/*.spec", '%{VERSION}')
-                            new_release = query_rpmspec("packages/**/${package_name}/*.spec", '%{RELEASE}')
+                            new_version = query_rpmspec("${spec_path}", '%{VERSION}')
```

Comment:
Do you need to interpolate the string here or could you simply us `spec_path` here?